README.md

```diff
@@ -1421,6 +1421,10 @@ Most often only these few lines in *gradle.properties* files are required to hav
 
 ```ini
 localInstance.backup.uploadUrl=sftp://example.com/aem/packages
+
+# Default suffix is .backup.zip. If you have other suffix and this property is not set, remote backup will not work.
+localInstance.backup.suffix=Backup.zip
```

Comment:
GAP examples are trying to highligh conventions to follow paradigm convention over configuration

you happily found it / that property... but when you hadn't followed to convention when worked on backups created by GAP so that it is a consequence of your previous mistake and I am ok with it.

generally speaking, this suffix guarantees that backups created anywhere by GAP will be later compatible with other, because both GAP builds will use same suffix. when you will change it, your backups will be not compatible with other builds so that ... build configuration / even examples should not encourage to do that.

Reply:
you have to have good reason to update it :) but it should not be documented here.